Gem/Code/Source/XRRPIExampleComponent.cpp

```diff
@@ -37,13 +39,18 @@ namespace AtomSampleViewer
 
     void XRRPIExampleComponent::Activate()
     {
+        if (m_xrSystem = AZ::RPI::RPISystemInterface::Get()->GetXRSystem();
+            m_xrSystem == nullptr)
+        {
+            return;
+        }
```

Comment:
Can you add this check to the XR RHI sample as well please? `AtomSampleViewer\Gem\Code\Source\RHI\XRExampleComponent.cpp`

Reply:
Done.